blocks/init/custom/accordion-item/accordion-item.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Template for the Service Box Block.
+ *
+ * @since 1.0.0
+ * @package Eightshift_Boilerplate\Blocks.
+ */
+
+namespace Eightshift_Boilerplate\Blocks;
+
+$title   = $attributes['title'] ?? '';
+$content = $attributes['content'] ?? '';
+
+$id              = $attributes['id'] ?? '';
+$block_class     = $attributes['blockClass'] ?? '';
+$block_js_class  = $attributes['blockJsClass'] ?? '';
```

Comment:
```suggestion
$block_js_class = $attributes['blockJsClass'] ?? '';
```

The above statements will have to be aligned